features/issues/components/IssueList/Issue-list.tsx

```diff
@@ -108,50 +120,75 @@ export function IssueList() {
   const { items, meta } = issuesPage.data || {};
 
   return (
-    <>
+    <Container>
       <Filters />
-      <Container>
-        <Table>
-          <thead>
-            <HeaderRow>
-              <HeaderCell>Issue</HeaderCell>
-              <HeaderCell>Level</HeaderCell>
-              <HeaderCell>Events</HeaderCell>
-              <HeaderCell>Users</HeaderCell>
-            </HeaderRow>
-          </thead>
-          <tbody>
+      <BottomContainer>
+        {isMobileScreen ? (
```

Comment:
Out of curiosity: Did you try a pure CSS solution?

Reply:
I didn't. My reasoning was that the table-based layout was very different than a smaller screen card component, so separating them entirely would be less complicated.